include/swift/Subsystems.h

```diff
@@ -276,6 +283,20 @@ namespace swift {
   bool performLLVM(IRGenOptions &Opts, ASTContext &Ctx,
                    llvm::Module *Module);
 
+  /// Run the LLVM passes. In multi-threaded compilation this will be done for
+  /// multiple LLVM modules in parallel.
```

Comment:
Please document the parameters here; at the very least it's important to note which of them can be null.